utils/common/src/main/java/brooklyn/util/ssh/IptablesCommands.java

```diff
@@ -89,6 +89,38 @@ public static String iptablesServiceStatus() {
         return iptablesService("status");
     }
 
+    @Beta // implementation not portable across distros
+    public static String firewalldService(String cmd) {
+        return sudo(alternatives(
+                BashCommands.ifExecutableElse1("systemctl", "systemctl " + cmd + " firewalld"),
+                "/usr/bin/systemctl " + cmd + " firewalld"));
```

Comment:
It's just as a precaution in case someone changed or removed environment variables for non-interactive shells